clearpath_sensors/package.xml

```diff
@@ -17,6 +17,7 @@
 
   <exec_depend>depthai_ros_driver</exec_depend>
   <exec_depend>ffmpeg_image_transport</exec_depend>
+  <exec_depend version_gte="8.0.0">fixposition_driver_ros2</exec_depend>
```

Comment:
Is this going to cause issues with the buildfarm? This package is only available from source, so we might need to remove this dependency. I hadn't considered this when I initially made the change in a feature branch.